mrp_bom_product_detail/__init__.py

```diff
@@ -20,4 +20,4 @@
 #
 ##############################################################################
 
-import mrp_bom_product_details
```

Comment:
What @elicoidal means is you should use short header format according to OCA guidelines. See https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py header's as an example. Pay attention to first 3 lines.